mirrord/kube/src/api.rs

```diff
@@ -36,9 +31,9 @@ where
     }
 }
 
-pub(crate) fn wrap_raw_connection(
+pub fn wrap_raw_connection(
```

Comment:
Warning: missing docs.

Even though it was missing before, would  you mind adding docs to this?